tests/ImageSharp.Tests/IO/ChunkedMemoryStreamTests.cs

```diff
@@ -13,6 +13,8 @@ namespace SixLabors.ImageSharp.Tests.IO;
 /// </summary>
 public class ChunkedMemoryStreamTests
 {
+    private readonly Random bufferFiller = new();
```

Comment:
```suggestion
    private readonly Random bufferFiller = new(123);
```

Setting a seed makes the test run deterministic, which I personally prefer.

Reply:
Yeah. Good point.